bears/general/IndentationBear.py

```diff
@@ -46,8 +47,12 @@ def run(self,
         :param tab_width:
             No. of spaces to insert for indentation.
             Only Applicable if use_spaces is False.
+        :param coalang_dir:
+            Full path of external directory containing the coalang
+            file for language.
         """
-        lang_settings_dict = LanguageDefinition(language)
+        lang_settings_dict = LanguageDefinition(
+                            language, coalang_dir=coalang_dir)
```

Comment:
well, if you add that commit before this one, then it still helps with 
testing, not?